finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,70 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Syntax Syd";
+var specialAbility = "resolve syntax errors";
 // Declare two variables - greeting AND catchphrase
+var greeting = ("Hello! I am " + heroName + "!");
+var catchphrase = `"Stop experiencing terrors, I am here to " ${specialAbility}`;
+
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
 // Declare two variables - power AND energy - set to integers
+var power = 3500;
+var energy = 7323;
+
+var fullPower = (power * 500);
```

Comment:
Similar to above - not convention to use `()` but it works, and some people will do that. Just really an FYI so you know it's not required and as you get into group projects you'll want all files in a project to be consistent.